ui/tests/specs/globalRules.spec.ts

```diff
@@ -2,16 +2,73 @@ import { test, expect } from "@playwright/test";
 
 const REGISTRY_UI_URL: string = process.env["REGISTRY_UI_URL"] || "http://localhost:8888";
 
+test.describe.configure({ mode: "parallel" });
```

Comment:
I see that you disabled some parallelism in the configuration: `fullyParallel: false`, so I'm wondering how this will play out.